### PR TITLE
fix#939-sec-analytics

### PR DESCRIPTION
**Pull request by sbcd90**
Signed-off-by: Subhobrata Dey <sbcd90@gmail.com>

### Description
fix#939-sec-analytics

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/install/plugins.md [hunk 1]
```diff
@@ -226,7 +226,7 @@ The following plugins are bundled with all OpenSearch distributions except for m
 | Performance Analyzer | [opensearch-performance-analyzer](https://github.com/opensearch-project/performance-analyzer) | 1.0.0 or newer |
 | Security | [opensearch-security](https://github.com/opensearch-project/security) | 1.0.0 or newer |
 | SQL | [opensearch-sql](https://github.com/opensearch-project/sql) | 1.0.0 or newer |
-| Security Analytics | [opensearch-security](https://github.com/opensearch-project/security-analytics) | 2.4.0 or newer |
+| Security Analytics | [opensearch-security-analytics](https://github.com/opensearch-project/security-analytics) | 2.4.0 or newer |
```
Comment (cwillum) [inline _opensearch/install/plugins.md:229]: From:
"| 2.4.0 or newer |"
Delete "or newer"

**Comment by cwillum** [comment]
@here To support fast-tracking Security Analytics documentation, I'm approving this PR. But I can see this PR is running into branch conflicts with plugins.md content. I believe another tech writer made changes to this file for the 2.4 release. So those changes will need to be taken into consideration when merging final changes in this PR. As of 11/17/2022, correct changes for the install/plugins page can be found here:
https://opensearch.org/docs/latest/opensearch/install/plugins/

(There is a note for Performance Analyzer at the bottom of the Plugins table, and we should be careful this doesn't disappear with these changes. Along with that, we can delete "or newer" from 2.4.0 in the Security Analytics entry in the table.)

**Review by cwillum — APPROVED** [review]
OK

**Comment by Naarcha-AWS** [comment]
@cwillum: I went ahead and fixed the merge conflict for you.

**Pull request merged**